stock_analysis/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analysis
+==============
+
+This module allows you to analyse your current stock.
```

Comment:
Why only current ;)?=>
This module extends the stock reporting to allow better stock per location analysis.